src/commands/server/update.ts

```diff
@@ -155,7 +156,7 @@ export default class Update extends Command {
             || cheCluster.spec.database.postgresImage
             || cheCluster.spec.server.pluginRegistryImage
             || cheCluster.spec.auth.identityProviderImage) {
-            cli.warn(`Eclipse Che operator won't update some components since their images are defined
+            cli.warn(`Eclipse Che operator won't be able to update some components since their images are defined
```

Comment:
Since we remove custom images from the CR we have to warn user.
```
To update Eclipse Che the images defined in the '${cheCluster.metadata.name}' Custom Resource of the namespace '${flags.chenamespace}' are going to be removed:
<images>
```
BTW
`if (!flags['skip-version-check'] && flags.installer === 'operator') {` -> `if (!flags['skip-version-check']) {`

Reply:
Done.

Reply:
Sorry, the message is not updated.